code/game/objects/items/grenades/grenade.dm

```diff
@@ -94,8 +94,8 @@
  * Grenades with other triggers like remote igniters probably skip this step and go straight to [/obj/item/grenade/proc/detonate]
  */
 /obj/item/grenade/proc/arm_grenade(mob/user, delayoverride, msg = TRUE, volume = 60)
-	var/turf/source_terf = get_turf(src)
-	log_grenade(user, source_terf) //Inbuilt admin procs already handle null users
+	var/turf/turf = get_turf(src)
+	log_grenade(user, turf) //Inbuilt admin procs already handle null users
```

Comment:
```suggestion
	log_grenade(user, get_turf(src)) //Inbuilt admin procs already handle null users
```

i said this in the comment :NOOO:

Reply:
@Mothblocks I was confused because /turf/turf was called in other places, like in a few lines above